### PR TITLE
vimPlugins: add localvimrc

### PR DESCRIPTION
**Pull request by erikryb**
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/misc/vim-plugins/default.nix [hunk 1]
```diff
@@ -584,6 +584,17 @@ rec {
 
   };
 
+  localvimrc = buildVimPluginFrom2Nix { # created by nix#NixDerivation
```
Comment (jagajaga) [inline pkgs/misc/vim-plugins/default.nix]: That was really created by vim2nix? Aren't there any github mirrors?

Comment (erikryb) [inline pkgs/misc/vim-plugins/default.nix]: That was generated by using vim-plugin-names-to-nix. There is a github mirror [here](https://github.com/embear/vim-localvimrc), but I don't know how to generate that instead. To be honest, I am not sure how vim2nix finds the sources from the plugin name.

File: pkgs/misc/vim-plugins/vim-plugin-names [hunk 2]
```diff
@@ -101,6 +101,7 @@
 "github:wakatime/vim-wakatime"
 "github:wincent/command-t"
 "goyo"
+"localvimrc"
```
Comment (jagajaga) [inline pkgs/misc/vim-plugins/vim-plugin-names]: Just change this to `github:embear/vim-localvimrc`. And re-run script.

**Comment by jagajaga** [comment]
Please open PR and do force push. Do not create additional PR.

File: pkgs/misc/vim-plugins/vim-plugin-names (hunk 2 above)
Comment (erikryb) [inline pkgs/misc/vim-plugins/vim-plugin-names]: Right. Didn't know github was prefered. I have fixed the commit.

**Pull request merged**